quotalib/qutoalib.go

```diff
@@ -9,12 +9,38 @@ import (
 	"io/ioutil"
 	"bytes"
 	"encoding/json"
+	"../types"
 )
 
+type QInfos struct{
```

Comment:
All the new type defines needs to be commented properly.  Add comments on the whole strucutre as well as individual variables